manifests/template/postgres-db.yaml.raw

```diff
@@ -78,10 +78,10 @@ spec:
       resources:
         limits:
           cpu: '{{.spec.resources.limits.cpu | default .spec.cpu | default "1000m"}}'
-          memory: '{{.spec.resources.limits.memory | default "512Mi"}}'
+          memory: '{{.spec.resources.limits.memory | default .spec.memory | default "512Mi"}}'
```

Comment:
These changes should be made here https://github.com/flanksource/template-operator-library/blob/main/config/templates/postgresql-db.yaml and then imported using the `make templates/postgresql-db.yaml`

Reply:
Ah cool. I will do that. This change must have been introduced after I started working on this issue. Now I see the header of the file mentioned that this was a generated file.